### PR TITLE
Use separate test environment

### PR DESCRIPTION
**Pull request by devmotion**
Since we dropped Julia < 1.3, we can use the [new way of specifying test dependencies on Julia >= 1.2](https://julialang.github.io/Pkg.jl/v1/creating-packages/#Test-specific-dependencies-in-Julia-1.2-and-above-1).

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -24,7 +24,7 @@ docs/.jekyll-cache
 
 .vscode
 .DS_Store
-Manifest.toml
-test/Project.toml
+/Manifest.toml
```
Comment (yebai) [inline .gitignore:27]: why switching to root level directory?

**Review by yebai — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (devmotion) [reply .gitignore:27]: True, we don't have to do that here. It's mostly useful for packages where the documentation is built with a specific reproducible set of packages which requires to add a `docs/Manifest.toml` file.

Comment (yebai) [reply .gitignore:27]: I see - thanks. Maybe keep both?

Comment (devmotion) [reply .gitignore:27]: Do you mean both `Manifest.toml` and `/Manifest.toml`? `test/Project.toml` is needed for specifying the test environment.

Comment (yebai) [reply .gitignore:27]: yes

**Pull request merged**